### PR TITLE
Feature/show done stories

### PR DESCRIPTION
**Pull request by jprask**
Show stories on the done column. [#24007](https://central.cm42.io/projects/central-board-v2#story-24007)

Before:

before

After:

after

### CODE REVIEW

File: spec/javascripts/actions/pastIterations_spec.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      PastIteration = {
+        getStories: sinon.stub().returns(stories)
+      }
+    });
+
+    it('calls dispatch with requestPastSotires', async () => {
+      const getState = sinon.stub();
+      getState.returns({ project, pastIterations });
+      const dispatch = sinon.stub();
+      dispatch.resolves({});
```
Comment (kostadriano) [inline spec/javascripts/actions/pastIterations_spec.js]: If you want to, you can do this in one line :).
```suggestion
      const dispatch = sinon.stub().resolves({});
```

Comment (jprask) [reply spec/javascripts/actions/pastIterations_spec.js]: Nice :+1:

File: app/assets/javascripts/models/beta/pastIteration.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { deserialize } from './story';
+
+export const getStories = async (projectId, startDate, endDate) => {
+    const params = serialize({startDate, endDate});
+    const { data } = await httpService
+        .get(`/project_boards/${projectId}/iterations`, { params });
+    return data.stories.map(story => deserialize(story.story));
+};
+
+export const pastIterationPropTypesShape = PropTypes.shape({
```
Comment (marcelobalexandre) [inline app/assets/javascripts/models/beta/pastIteration.js]: I could not find where this is being used. :thinking:

Comment (jprask) [reply app/assets/javascripts/models/beta/pastIteration.js]: Good catch! It's supposed to be used for the sprint component's `propTypes` but i forgot about it

**Pull request merged**